packages/material-ui-lab/src/Pagination/Pagination.js

```diff
@@ -171,10 +181,12 @@ Pagination.propTypes = {
    *
    * @param {PaginationRenderItemParams} params The props to spread on a PaginationItem.
    * @returns {ReactNode}
+   * @default (item) => <PaginationItem {...item} />
    */
   renderItem: PropTypes.func,
   /**
    * The shape of the pagination items.
+   * @default 'circular'
```

Comment:
Should we continue on the convention with `@returns` and `@param` to add a blank line between the description and the tags?
```suggestion
   *
   * @default 'circular'
```

Reply:
Not sure what the point would be. As far as I know every JSDOC parser returns the same result. TSDOC (which even has a formal spec) also does not care.

Reply:
The only advantage I can think about consistency for the developers reading our source. Blank space was hopefully added, in the first place, to make it easier to read, does that hold? I guess it's only a matter of getting used to it.

Reply:
Isn't this the opposite of consistency? Why add a newline after the description but not between tags?

Reply:
Is this a genuine argument that people are confused if sometimes theres a newline and sometimes there isn't?

Reply:
No specific preference, happy to remove all the spaces. I think that as long as it's predictable, we will get used to it and be able to scan the information faster.

Reply:
The big highlighted `@tagName` is not sufficient? Why are you not capable of distinguishing description from the following tags but able to distinguish the other tags?

Reply:
> that people are confused if sometimes theres a newline and sometimes there isn't?

I believe it relies on the same psychological leverages than prettier. I'm not an expert in this field so I can only relate to my personal experience. I would see my brain wire models to simplify the information of the word. The more patterns there are, the more efficient these models seem to be. Changing these models seem hard. 

On a related note, I'm amazed by the gestalt theory and how our brain simplify things, there is this great video about the theory I have just watched https://youtube.com/watch?v=FryaH599ec0.

Reply:
prettier doesn't add newlines between statements though.

Reply:
Again: What makes description so special and how do you reconcile the additional newline with missing newlines between tags?

Reply:
No specific preference, We could remove all the blank lines (between description and returns, param tags). We could consider descriptions and tags as two different piece of information and add a blank line, etc. All of that seems arbitrary.

At the end of the day, I think that following one rule is enough. This rule can be anything as long as we can easily apply and understand it.